roles/libvirt_manager/molecule/boot_order/converge.yml

```diff
@@ -0,0 +1,238 @@
+             prefix='24'>
+            </ip>
+          </network>
+  tasks:
+    - name: Load networking definition
+      ansible.builtin.include_vars:
+        file: input.yml
+        name: cifmw_networking_definition
+
+    - name: Deploy layout with boot_order configurations
+      ansible.builtin.import_role:
+        name: libvirt_manager
+        tasks_from: deploy_layout
+
+    - name: Verify boot_order configurations
+      block:
+        # Test 1: Verify disk-first VM has correct boot order
+        - name: Get disk_first VM XML
+          register: _disk_first_xml
+          community.libvirt.virt:
+            command: "get_xml"
+            name: "cifmw-disk-first-0"
+            uri: "qemu:///system"
+
+        - name: Check disk boot order in disk-first VM
+          register: _disk_first_disk_boot
+          community.general.xml:
+            xmlstring: "{{ _disk_first_xml.get_xml }}"
+            xpath: "/domain/devices/disk[@device='disk']/boot"
+            content: "attribute"
+
+        - name: Check interface boot order in disk-first VM
+          register: _disk_first_net_boot
+          community.general.xml:
+            xmlstring: "{{ _disk_first_xml.get_xml }}"
+            xpath: "/domain/devices/interface[1]/boot"
+            content: "attribute"
+
+        - name: Assert disk-first VM has correct boot order
+          ansible.builtin.assert:
+            that:
+              - _disk_first_disk_boot.matches[0].boot.order == "1"
+              - _disk_first_net_boot.matches[0].boot.order == "2"
+            quiet: true
+            msg: >-
+              Expected disk boot order=1 and network boot order=2,
+              got disk={{ _disk_first_disk_boot.matches[0].boot.order }}
+              and network={{ _disk_first_net_boot.matches[0].boot.order }}
+
+        # Test 2: Verify network-first VM has correct boot order
+        - name: Get net_first VM XML
+          register: _net_first_xml
+          community.libvirt.virt:
+            command: "get_xml"
+            name: "cifmw-net-first-0"
+            uri: "qemu:///system"
+
+        - name: Check disk boot order in network-first VM
+          register: _net_first_disk_boot
+          community.general.xml:
+            xmlstring: "{{ _net_first_xml.get_xml }}"
+            xpath: "/domain/devices/disk[@device='disk']/boot"
+            content: "attribute"
+
+        - name: Check interface boot order in network-first VM
+          register: _net_first_net_boot
+          community.general.xml:
+            xmlstring: "{{ _net_first_xml.get_xml }}"
+            xpath: "/domain/devices/interface[1]/boot"
+            content: "attribute"
+
+        - name: Assert network-first VM has correct boot order
+          ansible.builtin.assert:
+            that:
+              - _net_first_net_boot.matches[0].boot.order == "1"
+              - _net_first_disk_boot.matches[0].boot.order == "2"
+            quiet: true
```

Comment:
wondering why everywhere is `quiet: true`

Reply:
:) Honestly I don't know, I (or we) used what exists in https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/libvirt_manager/molecule/ocp_layout/converge.yml#L146 as inspiration. And it has `quiet: true`.